src/main/js/helpers/constants.js

```diff
@@ -53,6 +53,22 @@ export const COLORS = {
     LAVENDER: "#8374c2",
     WHITE: "#ffffff"
 };
+/**
+ *
```

Comment:
Can you please add small description on these color palettes.

Reply:
Added here:
https://github.com/cerner/carbon-graphs/pull/308/commits/a5a66a75beca5bc9ed5a4f706e9ea53dfa9a68e3